modules/messages/tests/server/message.server.routes.tests.js

```diff
@@ -380,7 +380,7 @@ describe('Message CRUD tests', function () {
                     '<br />blockquote' +
                     '<p>' +
                     '<a href="https://www.trustroots.org/">link</a>' +
-                    '<a href="http://www.trustroots.org">www.trustroots.org</a>' +
+                    '<a href="https://www.trustroots.org">www.trustroots.org</a>' +
```

Comment:
This test fails: the original link `www.trustroots.org` actually formats to `<a href="http://www.trustroots.org">www.trustroots.org</a>`

(to http)

So this change should be reverted, in order for tests to pass again.